package.json

```diff
@@ -5,8 +5,8 @@
     "main": "lib/index.js",
     "types": "lib/index.d.ts",
     "dependencies": {
-        "long": "4.0.0",
-        "@types/long": "4.0.1"
+        "@types/long": "4.0.1",
```

Comment:
These dependencies' order changes every time I run npm install. So I've kept it like this.